internal/service/s3control/multi_region_access_point_data_source.go

```diff
@@ -0,0 +1,92 @@
+package s3control
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/s3control"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-aws/internal/conns"
+	"github.com/hashicorp/terraform-provider-aws/internal/verify"
+)
+
+func DataSourceMultiRegionAccessPoint() *schema.Resource {
+	return &schema.Resource{
+		ReadWithoutTimeout: dataSourceMultiRegionAccessPointBlockRead,
+
+		Schema: map[string]*schema.Schema{
```

Comment:
Drive-by cosmetic comment 💄  - preferably we order the schema alphabetically.

Reply:
I have ordered the schema order like this because the `account_id` and `name` are both parameters that are needed as argument for the data source. From `alias` down are attributes that are returned when using the data source.